temporal-opentracing/src/main/java/io/temporal/opentracing/SpanOperationType.java

```diff
@@ -34,7 +34,10 @@ public enum SpanOperationType {
   UPDATE_WORKFLOW("UpdateWorkflow"),
   HANDLE_QUERY("HandleQuery"),
   HANDLE_SIGNAL("HandleSignal"),
-  HANDLE_UPDATE("HandleUpdate");
+  HANDLE_UPDATE("HandleUpdate"),
+  EXECUTE_NEXUS_OPERATION("ExecuteNexusOperation"),
+  START_NEXUS_OPERATION("StartNexusOperation"),
+  CANCEL_NEXUS_OPERATION("CancelNexusOperation");
```

Comment:
To match other span names, I wonder if this should be

```suggestion
  START_NEXUS_OPERATION("StartNexusOperation"),
  RUN_NEXUS_OPERATION("RunNexusOperation"),
  CANCEL_NEXUS_OPERATION("CancelNexusOperation");
```

It seems we use start for outbound things and run for inbound things. I would usually not consider this a blocker, but we copy these names in the other SDKs too.

Reply:
Yes I was inconsistent here since the handler side is called `startOperation`

Reply:
in the future we will have types like `GetResult` or `GetInfo` so I think the types here should reflect the method being called

Reply:
But unlike activities the Nexus operation is not being "Run" in this span, it is simply being started. if we wanted to keep the  `Run` prefix I would be OK with `StartNexusOperation ` -> `RunStartNexusOperation `

Reply:
I think the same thing happens to child workflows though, especially with sticky disabled, and especially in SDKs that do not recreate this span during replay. The operation is being run, it just may say it's not completing yet. I think we would have to use at least `RunStartNexusOperation` since `StartNexusOperation` is for the caller side.